contents/docs/contribute/recognizing-contributions.md

```diff
@@ -8,9 +8,9 @@ At PostHog we aim to recognize all contributions made to our open source codebas
 
 ## Pull requests
 
-If you submit a Pull Request to a repository under the `PostHog` organization, a bot will automatically try to send you an email with a merch code. If this fails, the bot will try to let you know in a comment so that you can email Yakko (the bot creator and babysitter) and get your merch code. 
+If you submit a Pull Request to a repository under the `PostHog` organization, a bot will automatically send you an email to get in touch with us for a merch code. 
 
-If, after a few days of having had your PR merged, you still didn't get a merch code, you can email us at _[hey@posthog.com](mailto:hey@posthog.com)_ and we'll sort it out!
+We usually give out $100 for a product or app-related PR, and $25 for something on posthog.com (but we have gone over and above this for particularly large community PRs before!)
```

Comment:
Is this what we'll be doing from now on?

Bot was set to 30$/50$ initially but unsure if anyone's worked on it since

Reply:
This is just my suggestion. My understanding is that the bot doesn't actually send a code now though as it seems to just point people to email hey@? (At least that's based on all the emails we've been receiving saying 'I got an email telling me to contact you'.

Reply:
<img width="931" alt="Screenshot 2022-08-25 at 11 44 03" src="https://user-images.githubusercontent.com/70321811/186644725-ba01be30-f4b0-47e9-86ca-cffd5b7decc8.png">